new-components/scanners/bandit/docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+services:
```

Comment:
In the .env I see that we rely on postgres but I don't see a postgres container here 🤔

Reply:
i just copied the example .env, the gosec one also doesn't have any env vars and has the postgresql things